### ISSUE TITLE
Authorize before fetching data

### ISSUE THREAD

**Issue by alexander37137** [palkan/action_policy#82]
Can i change authorize behavior to check policy before fetching data?

**Comment by palkan** [comment]
Are you talking about GraphQL integration (and here #81 as well)?

**Comment by alexander37137** [comment]
Yes, I want to make authorization api using graphql and i have to make graphql controller available without authorization. And that make problems with other apis. I made user nullable in graphql context and after that everywhere in queries i have to check presence of current_user
